dashboard/app/controllers/api/v1/user_school_infos_controller.rb

```diff
@@ -0,0 +1,75 @@
+class Api::V1::UserSchoolInfosController < ApplicationController
+  before_action :authenticate_user!
+  before_action :load_user_school_info
+
+  skip_before_action :verify_authenticity_token
```

Comment:
why don't we care about authenticity tokens here?

Reply:
The authenticity token is required when a user views a form to create, update or destroy a resource (include link). 
 When a user clicks on the 'Yes' button in the Confirmation Dialog, user does not see a form.